packages/twitter-verifier-circuits/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@twitter-verifier/circuits",
-  "version": "1.0.1",
+  "version": "2.0.0",
   "scripts": {
     "test": "NODE_OPTIONS=--max_old_space_size=16384 jest"
   },
   "dependencies": {
     "@zk-email/circuits": "workspace:^",
     "@zk-email/helpers": "workspace:^",
     "big-integer": "^1.6.51",
-    "snarkjs": "https://github.com/sampritipanda/snarkjs.git#fef81fc51d17a734637555c6edbd585ecda02d9e"
+    "snarkjs": "git+https://github.com/vb7401/snarkjs.git#24981febe8826b6ab76ae4d76cf7f9142919d2b8"
```

Comment:
Wait this doesn't work well with the frontend; sampritipanda's fork was needed for that. Are you sure this works?

Reply:
This is only twitter-circuits package.json where we use snarkjs for zkey generation and all. We are using vb fork anyway in the [sh file](https://github.com/zkemail/zk-email-verify/blob/main/packages/twitter-verifier-circuits/scripts/3_gen_both_zkeys.sh#L22) when generating chunked zkeys. Just made that as the default in package.json for this package. Its same either way as the scripts override to use this when needed.
Can revert if it dont make sense. FE works fine.

Reply:
Huh weird. I don't remember why we use vb7401 to generate chunked zkeys instead of sampritipanda. Should be fine then I guess.

Reply:
👍 
Hmm, next time we need to change circuit, I will compare the commits.